add note about impersonation coming soon (#23)
workos/authkit-react: README.md

* `organizationId`: The currently-selected organization.
* `role`: The `role` of the user for the current organization.
* `permissions`: Permissions corresponding to this role.

> ## Impersonation support is coming soon

Reviewer: ```suggestion
## Impersonation
```

> Impersonation is not currently supported in authkit-js. We are targeting EOY 2024.

Reviewer: ```suggestion
Impersonation is not currently supported in authkit-js but is on the roadmap for 2024.
```
